data/config/court.mainnet.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
     appealCollateralFactor:        bn(30000),                    // appeal collateral is 3x of the corresponding juror fees
     appealConfirmCollateralFactor: bn(20000),                    // appeal-confirmation collateral is 2x of the corresponding juror fees
     finalRoundWeightPrecision:     bn(1000),                     // use to improve division rounding for final round maths
+    skippedDisputes:               1,                            // number of dispute to skip
```

Comment:
Wondering on this; are we "skipping" disputes so that the new DisputeManager's events are still lined up with the old one?

Reply:
yes, and to avoid having disputes with same ID